packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-scopes/index.md

```diff
@@ -6,20 +6,20 @@ Scopes specify what access privileges are being requested as part of the authori
 
 If you need scopes in addition to the reserved scopes provided, you can create them. Custom scopes can have corresponding claims that tie them to some sort of user information.
 
-1. In the Admin Console, navigate to **Security > API**.
-2. On the **Authorization Servers** tab, select the name of the Authorization Server, and then select **Scopes**.
-3. Select **Scopes**, and then **Add Scope**.
-4. Enter a **Name**, **Display phrase**, and **Description**.
+1. In the Admin Console, go to **Security** > **API**.
+1. On the **Authorization Servers** tab, select the name of the Authorization Server, and then select **Scopes**.
+1. Select **Scopes**, and then **Add Scope**.
+1. Enter a **Name**, **Display phrase**, and **Description**.
 
     > **Note:** The **Display phrase** is what the user sees in the **Consent** dialog box.
 
-5. Select **Require user consent for this scope** to require that a user grant consent for the scope.
+1. Select **Require user consent for this scope** to require that a user grant consent for the scope.
 
     > **Note:** You can configure individual clients to ignore this setting and skip consent.
 
-6. Select **Set as a default scope** if you want to allow Okta to grant authorization requests to apps that don't specify scopes on an authorization request. If the client omits the scope parameter in an authorization request, Okta returns all default scopes in the access token that are permitted by the access policy rule.
-7. Select **Include in public metadata** if you want the scope to be [publicly discoverable](/docs/reference/api/oidc/#well-known-oauth-authorization-server).
-8. Click **Create**.
+1. Select **Set as a default scope** if you want to allow Okta to grant authorization requests to apps that don't specify scopes on an authorization request. If the client omits the scope parameter in an authorization request, Okta returns all default scopes in the access token that are permitted by the access policy rule.
```

Comment:
Up to you, but I think you can take out "to allow" and just have "...if you want Okta to grant authorization requests..."

Reply:
Changed to:
> Select **Set as a default scope** if you want Okta to grant authorization requests to apps that don't specify scopes on an authorization request. If the client omits the scope parameter in an authorization request, Okta returns all of the default scopes that are permitted in the access token by the access policy rule.